bpf/bpf_overlay.c

```diff
@@ -333,6 +339,55 @@ int tail_handle_ipv4(struct __ctx_buff *ctx)
 					      CTX_ACT_DROP, METRIC_INGRESS);
 	return ret;
 }
+
+#ifdef ENABLE_VTEP
+/*
+ * ARP responder for ARP requests from VTEP
+ * Respond to remote VTEP endpoint with cilium_vxlan MAC
+ */
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_ARP)
+int tail_handle_arp(struct __ctx_buff *ctx)
+{
+	union macaddr mac = NODE_MAC;
+	union macaddr smac;
+	__be32 sip;
+	__be32 tip;
+	__u32 monitor = 0;
```

Comment:
One minor nit here, if monitor is 0 then it's off. This should be `TRACE_PAYLOAD_LEN`.

```suggestion
	__u32 monitor = TRACE_PAYLOAD_LEN;
```